example/android/app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think we can probably also gitignore these AndroidManifest.xml files in /debug and /profile as I think they are generated, but it's probably also not an issue to commit them